graylog2-web-interface/src/stores/configurations/ConfigurationsStore.js

```diff
@@ -115,6 +115,24 @@ const ConfigurationsStore = Reflux.createStore({
     ConfigurationActions.update.promise(promise);
   },
 
+  updateWhitelist(configType, config) {
+    const promise = fetch('PUT', URLUtils.qualifyUrl('/system/urlwhitelist'), config);
+
+    promise.then(
+      () => {
+        this.configuration[configType] = config;
+        this.propagateChanges();
+        UserNotification.success('Configuration updated successfully');
+        return config;
+      },
+      (error) => {
+        UserNotification.error(`Search config update failed: ${error}`, `Could not update search config: ${configType}`);
```

Comment:
We should adjust the wording here to refer to the URL whitelist instead of "Search".

Reply:
sure ! Thank you for the catch.